### ISSUE TITLE
readme on pypi

### ISSUE THREAD

**Issue by knaaptime** [pysal/submodule_template#44]
i just noticed that with our current tooling, most of the packages lose their READMEs on pypi. spopt still has a full description, but spaghetti, tobler, momepy, spreg all just have a title. It's not the worst thing in the world, but probably an easy fix

**Comment by martinfleis** [comment]
Yeah, super easy. See https://github.com/martinfleis/clustergram/blob/main/pyproject.toml

**Comment by martinfleis** [comment]
We just need to ensure we don't use unsupported formatting. Pypi is a bit more restrictive about some characters from what I remember.

**Comment by knaaptime** [comment]
ah, funny. We just need to specify the readme file in the `readme` line of toml?

**Comment by martinfleis** [comment]
Yeah
